From 06d3a7c0034c0f9a28c5dcd5a6a7b2fad817316b Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Tue, 13 Jul 2021 02:19:52 -0700
Subject: [PATCH] Fix build with qt-6

* fixes:
| ../git/main.cpp: In function 'QString readSmackLabel()':
| ../git/main.cpp:27:12: error: variable 'QFile f' has initializer but incomplete type
|    27 |     QFile f("/proc/self/attr/current");
|       |            ^
| ../git/main.cpp:28:23: error: incomplete type 'QFile' used in nested name specifier
|    28 |     if (f.open(QFile::ReadOnly | QFile::Text)) {
|       |                       ^~~~~~~~
| ../git/main.cpp:28:41: error: incomplete type 'QFile' used in nested name specifier
|    28 |     if (f.open(QFile::ReadOnly | QFile::Text)) {
|       |                                         ^~~~

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 main.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index f0d71cf..1f4e9d1 100644
--- a/main.cpp
+++ b/main.cpp
@@ -18,6 +18,7 @@
 #include <QQmlApplicationEngine>
 #include <QQmlContext>
 #include <QQuickView>
+#include <QFile>
 
 #define SMACK_LABEL     "webOS::App::com.webos.app.test.smack.native"
 
@@ -48,4 +49,4 @@ int main(int argc, char *argv[])
     view.show();
 
     return app.exec();
-}
\ No newline at end of file
+}
